genScript.js

```diff
@@ -0,0 +1,61 @@
+const services = require('./services')
+
+console.log('#!/bin/bash')
+console.log('set -ex')
+
+switch (process.argv.pop()) {
+  case 'checkout':
+    for (const service of services) {
+      console.log(`git clone ${service.repo} ${service.name}`)
```

Comment:
I'd be a bit worried about shell injection here, _but_, I doubt we'll ever have data that's not under our control, so...

Reply:
Yeah, I do not think this is an issue: Both `services.js` and this script are in the same folder, so write access to one but not the other seems unlikely :wink: